### ISSUE TITLE
Intermittent 'System Value' API Call Failure

### ISSUE THREAD

**Issue by psdevil** [EDSM-NET/FrontEnd#452]
Hi Team,

I am getting the following errors when trying to get a system value using both the ID and the system name.

The URL I am using is in the error message. It looks like a host error as the website is not responding in time,

I have tried a number of system ID's and system names. Since I started raising this issue, it is now failing around 95% of the time so it is not limited to a specific name or ID.

This started around 18:15 BST today and is still persisting. Hopefully it is not the server being overloaded.

I am using Python v3.9.0 and have not any similar issues before today.

Feel free to ping me if you need more info,

EDIT: The same also is happening on other API calls. See the last example.

```
HTTPSConnectionPool(host='www.edsm.net', port=443): Max retries exceeded with url: /api-system-v1/estimated-value?systemId=28866288
(Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000012D66C13310>: Failed to establish a new connection: [WinError 10060] 
A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond')
```

```
HTTPSConnectionPool(host='www.edsm.net', port=443): Max retries exceeded with url: /api-system-v1/estimated-value?systemId=12967838
(Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001E679DE04F0>: Failed to establish a new connection: [WinError 10060]
A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))
```

```
HTTPSConnectionPool(host='www.edsm.net', port=443): Max retries exceeded with url: /api-system-v1/estimated-value?systemName=Mel+22+Sector+AQ-O+c6-6
(Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002ABE575F880>: Failed to establish a new connection: [WinError 10060]
A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))
```

```
HTTPSConnectionPool(host='www.edsm.net', port=443): Max retries exceeded with url: /api-v1/sphere-systems?systemName=Syneayoo+WK-F+d11-0&radius=50&minRadius=0&showId=1&showCoordinates=1&showPermit=1&showInformation=1&showPrimaryStar=1
(Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000213CAD0D3D0>: Failed to establish a new connection: [WinError 10060]
A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))
```

**Comment by AnthorNet** [comment]
Hey, just tested in my browser and did not have any trouble reaching the mentioned URL

**Comment by psdevil** [comment]
I reset my home network and still had the issues when I went to bed around 1am UK time, I reported the issue to my service provider who said they could see nothing wrong. They passed me to 2nd line who confirmed the issues I was seeing.

Spoke to a neighbour a little while ago and he had similar issues with other sites until around 2am. He's a IT support shift worker on nights this month. He reported his issues to the same service provider who told him around 4am that they were upgrading our local infrastructure and it shouldn't have affected us. Obviously it did. Our support calls are still open with the service provider as they are investigating our cases further.

Happy to close the issue as it plainly wasn't an issue at your end.
